coral/src/app/features/topics/details/documentation/TopicDocumentation.tsx

```diff
@@ -0,0 +1,32 @@
+import { PageHeader, Button, Box } from "@aivenio/aquarium";
+import { NoDocumentationBanner } from "src/app/features/topics/details/documentation/components/NoDocumentationBanner";
+import { useTopicDetails } from "src/app/features/topics/details/TopicDetails";
+import { DocumentationView } from "src/app/features/topics/details/documentation/components/DocumentationView";
+
+function TopicDocumentation() {
+  const { topicOverview } = useTopicDetails();
+
+  return (
+    <>
+      <PageHeader title={"Documentation"} />
+      {!topicOverview?.topicDocumentation ||
```

Comment:
Nitpick: I really think we should avoid boolean checks on `undefined`, I think we had a bug the other day precisely because of it ^^'

Reply:
I could do `Boolean(topicOverview.topicDocumentation)` -> that would cover the empty string, too. Which I could have thought of before :D

Reply:
I'm just thinking, why not `topicOverview?.topicDocumentation === undefined` ? It is exactly wgat we are checking ^^'

Reply:
I just think that type coercion in JS is not a good feature, and we should avoid it when possible ^^'

Reply:
I also need to check that the string is not empty, so it would be shorter :D

Reply:
It would, but it would obscure exactly what we are checking, and expose ourselves to wird bugs ^^ I don't think shorter is better in these instances. 

If we really want a short inline check, then we can asign the check to a variable like
```tsx
const hasDocumentation = topicOverview?.topicDocumentation !== undefined && topicOverview.topicDocumentation.length > 0
```

It's really not just tase, I strongly believe this is more legible and avoids bugs

Reply:
I can do that. I've to do it without the variable though, because TS isn't super smart sometimes 😅 
<img width="593" alt="Screenshot 2023-06-26 at 09 34 40" src="https://github.com/aiven/klaw/assets/943800/55a17e60-ac23-4111-ac64-ae838a59749a">